template.tpl

```diff
@@ -332,14 +340,12 @@ scenarios:
     ];
 
     mock('copyFromWindow', key => {
```

Comment:
This is now a duplicate of what you are doing bellow.

Reply:
Yes, but these are two separate tests